homeassistant/components/switch/scsgate.py

```diff
@@ -8,46 +8,44 @@
 
 import homeassistant.components.scsgate as scsgate
 from homeassistant.components.switch import SwitchDevice
-from homeassistant.const import ATTR_ENTITY_ID
+from homeassistant.const import (ATTR_ENTITY_ID, ATTR_STATE, CONF_NAME)
+
+ATTR_SCENARIO_ID = 'scenario_id'
 
 DEPENDENCIES = ['scsgate']
 
+CONF_TRADITIONAL = 'traditional'
+CONF_SCENARIO = 'scenario'
+
+PLATFORM_SCHEMA = scsgate.PLATFORM_SCHEMA
```

Comment:
The big reason for this is that the switch component could add extra variables to configure platforms in the future.